turn_by_turn/io.py

```diff
@@ -90,4 +93,20 @@ def write_tbt(output_path: Union[str, Path], tbt_data: TbtData, noise: float = N
     else:
         if noise is not None:
             tbt_data = add_noise_to_tbt(tbt_data, noise=noise, seed=seed)
-        return module.write_tbt(output_path, tbt_data)
+        return module.write_tbt(output_path, tbt_data, **additional_args(datatype))
+
+
+def additional_args(datatype: str) -> dict[str, Any]:
```

Comment:
Is it dict[str, Any], because of the empty dict? If not, wouldn't it be dict[str, str]?

Reply:
It doesn't really matter what the value is, as long as it is a value the reader/writer functions expect. It's more or less by chance that it is currently a string, but in case we ever implement a writer that expects something else, that is also possible with this function. So it is more a type-hint of an intent instead of the current situation.